app.rb

```diff
@@ -21,7 +21,11 @@
 
   begin
     parser = Fluent::TextParser::RegexpParser.new(Regexp.new(@regexp))
-    parser.configure('time_format' => @time_format) if not @time_format.empty?
+    unless @time_format.empty?
+      parser.configure(
+        Fluent::Config::Element.new('', '', { 'time_format' => @time_format }, [])
```

Comment:
Line is too long. [82/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.